q0_gui.py

```diff
@@ -143,7 +147,38 @@ def show_q0_data(self):
         while self.q0_fit_plot_items:
             self.q0_fit_plot.removeItem(self.q0_fit_plot_items.pop())
 
+        while self.selection_linear_items:
+            self.q0_data_plot.removeItem(self.selection_linear_items.pop())
+
         measurement = self.selectedCM.q0_measurement
+
+        def region_updated(region_item):
```

Comment:
Ok I'm slowly going through this and my first question is why this function is defined inside of `show_q0_data`? and when the `region_item` argument is being used

Reply:
Thanks, I missed removing `region_item`, it isn't being used anywhere. 

 I wasn't able to figure out a good way to pass as an argument when defined outside because of the `.connect()`, so I defined the `region_updated()` methods within `show_q0_data()` and `show_calibration_data()` since it needed the `self.selectedCM` variable to access data/runs

Reply:
Oh `self.selectedCM` is available everywhere within an object! So you should be able to pull it out just fine

Reply:
Ah right, I can move it within the Q0Gui class then. (I think I had tried to add it to a util file, which wouldn't work.) Thanks!